app/controllers/rooms_controller.rb

```diff
@@ -4,6 +4,7 @@ def new
 
   def create
     room = Room.new(room_params)
+    room.choices.build(title: params[:room][:choice])
```

Comment:
So same thing that I was mentioning on @AllPurposeName's PR yesterday. It would probably be smarter not to create associations between our models until they are saved. So this could be:
```ruby
room = Room.new(room_params)
choice = Choice.new(title: params[:room][:choice])
if room.save
    room.choices << choice
    ...
```